src/shared/analytics/analytics.background.ts

```diff
@@ -177,6 +185,7 @@ function trackAppEvents({ account }: { account: Account }) {
         ...addressActionAnalytics,
       });
       sendToMetabase('signed_transaction', params);
+      gaCollect(account, 'signed_transaction', params);
```

Comment:
may be we can introduce a helper `prepareGaParams(account, params)` that will get a userId from account and add it to params. To make google-analytics service independent from Account class implementation. Not a big deal though.